examples/storybook/src/wrap-with-hits.ts

```diff
@@ -62,6 +62,7 @@ export function wrapWithHits({
 }: WrapWithHitsParams) {
   @Component({
     selector: 'ais-app',
+    styles,
```

Comment:
so if this was just added here, where was it used before?

Reply:
[This](https://github.com/algolia/angular-instantsearch/pull/582/files/5ce77df55c6680469178c958de65d188bf1785f1#diff-765617f67bcb0ee3c2dfd175931764afL51) `styles` in `wrapWithHits` has never been used before.

Reply:
interesting that this didn't throw an error from linting :(

Reply:
I hope this kind of things get caught when we move onto the latest version of eslint-config-algolia.